ibex_core.core

```diff
@@ -54,6 +54,10 @@ parameters:
     datatype: bool
     paramtype: vlogparam
 
+  MultiplierImplementation:
+    datatype: str
+    paramtype: vlogparam
```

Comment:
Can you add a description and a default value to this parameter?

```
description: multiplier implementation. Valid values: fast, slow
default: fast
```

Reply:
BTW `description` requires quotation marks in this case.